### ISSUE TITLE
Figure Out Why Looker Times Out at 2 Minutes for SQL Explorer Queries

### ISSUE THREAD

**Issue by sheyd** [DoSomething/quasar#387]
_BUG_
-------------------
#### Current Behavior
Looker has times out around 2 minutes for queries run in SQL Runner.

#### Desired Behavior
Looker times out to a designated value we give it.

#### Steps to Replicate
Run 
```
 select
        `key` as 'key',
        `timestamp` as 'timestamp',
        `run_nid` as 'run_nid',
        `northstar_id` as 'northstar_id',
        `source` as 'source'
      from(
      select ### pre gambit mobile data ###
          CONCAT(mml.ms_phone_number, mml.mc_run_id) as 'key',
          mml.ms_activated_at as 'timestamp',
          mml.mc_run_id as 'run_nid',
          'pre-gambit' as 'source',
          qu.northstar_id as 'northstar_id' 
        from users_and_activities.mobile_master_lookup_lite mml
          left join quasar.users qu on qu.drupal_uid = mml.mu_uid
        union all 
          select ### rogue sms data ### 
            ca.signup_id as 'key',
            ca.signup_created_at as 'timestamp', 
            ca.campaign_run_id as 'run_nid', 
            ca.northstar_id as 'northstar_id',
            'rogue' as 'source'
            from quasar.campaign_activity ca
            where ca.signup_source = 'sms-mobilecommons' 
          ) as a
          group by a.key
```
#### Why This Matters
Blocking accurate MAM KPI.

**Comment by sheyd** [comment]
Found this Looker article: https://discourse.looker.com/t/query-stops-at-90-seconds/3795/10, which breaks down that the Looker front-end is the upper limit on the query timing. Ideally, it would be nice, if the UI just updated till a query was done, independent of a front-end status, but that's Looker's limitation for now. 

I've upped HA Proxy to 5 minutes from the 2 minutes it was before, and am re-testing query above.
